source/solvers/vof_algebraic_interface_reinitialization.cc

```diff
@@ -0,0 +1,835 @@
+// SPDX-FileCopyrightText: Copyright (c) 2020-2024 The Lethe Authors
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception OR LGPL-2.1-or-later
+
+#include <core/solutions_output.h>
+
+#include <solvers/vof_algebraic_interface_reinitialization.h>
+
+#include <deal.II/lac/solver_control.h>
+#include <deal.II/lac/sparsity_tools.h>
+#include <deal.II/lac/trilinos_solver.h>
+
+#include <sys/stat.h>
+
+template <int dim>
+void
+VOFAlgebraicInterfaceReinitialization<dim>::setup_dofs()
+{
+  // Get MPI communicator
+  auto mpi_communicator = this->triangulation->get_communicator();
+
+  // Distribute and renumber DoFs
+  this->dof_handler.distribute_dofs(*this->fe);
+  DoFRenumbering::Cuthill_McKee(this->dof_handler);
```

Comment:
Mostly a note for myself, but i am not sure we really need to renumber or if it really improves stuff